virttest/network.py

```diff
@@ -0,0 +1,57 @@
+import logging
+
+
+def get_channel_info(session, interface):
+    """
+    Get channel parameters of interface in vm.
+
+    :param session: the guest session
+    :param interface: the interface name shows on the guest, for example: eth0
+    :return: 2 dictionaries, one is the maximum values for hardware, the other is current value,
+    in format as {"RX": "n/a", "TX": "n/a", "Other": "n/a", "Combined": "1"}
+    """
+    cmd = "ethtool -l %s" % interface
+    s, o = session.cmd_status_output(cmd)
+    if s:
+        logging.info("Get channel parameters for vm failed:%s" % o)
```

Comment:
It seems you forgot to update it to 'error' :D

Reply:
updated